LICENSE

```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```

Comment:
Don't we need the explicit agreement of all previous  (major) contributors to change a software license ?